tests/telemetry/test_telemetry.py

```diff
@@ -19,21 +25,25 @@ def get_list_stdout(cmd_out):
         out_list.append(result)
     return out_list
 
+def setup_telemetry_forpyclient(duthost):
+    """ Set client_auth=false. This is needed for pyclient to sucessfully set up channel with gnmi server.
+        Restart telemetry process
+    """
+    set_client_auth = duthost.shell('/usr/bin/redis-cli -n 4 hset "TELEMETRY|gnmi" "client_auth" "false"', module_ignore_errors=False)
```

Comment:
should check if client_auth is already false, if it is then no need to restart the service.

Reply:
updated logic to check if client_auth is false or not